allink_core/core_apps/allink_image_svg/models.py

```diff
@@ -0,0 +1,51 @@
+from cms.models import CMSPlugin
+from django.db import models
+
+from django.core.exceptions import ValidationError
+from django.utils.safestring import mark_safe
+import re
+
+from filer.fields.file import FilerFileField
+
+from allink_core.core.models import AllinkLinkFieldsModel
+
+
+class AllinkImageSVGPlugin(AllinkLinkFieldsModel, CMSPlugin):
+    picture = FilerFileField(
+        verbose_name='Image',
+        blank=True,
+        null=True,
+        on_delete=models.PROTECT,
+        related_name='%(app_label)s_%(class)s_image',
+    )
+
+    is_inline = models.BooleanField(
```

Comment:
what's the usecase for "non"-inlined svg images?

Reply:
If we want to manipulate svg styles with CSS, the logo needs to be inlined.